content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -19,136 +16,129 @@ deploy Kubeflow on Amazon Web Services (AWS).
-    curl https://raw.githubusercontent.com/kubeflow/kubeflow/${KUBEFLOW_TAG}/scripts/download.sh | bash
+    roles:
+      - eksctl-kubeflow-aws-nodegroup-ng-a2-NodeInstanceRole-xxxxxxx
     ```
 
-    * KUBEFLOW_SRC - Full path to your preferred download directory. Please use the full absolute path, for example `/tmp/kubeflow-aws`
-
 1. Run the following commands to set up your environment and initialize the cluster.
+    Since there're many ways to create your cluster, we highly recommend you to get our own eks cluster ready.
 
     > Note: If you would like to install Kubeflow on your existing EKS cluster, please skip this step
     > and follow the setup instructions for an [existing cluster](/docs/aws/deploy/existing-cluster) instead.
     > When you are finished, return here and resume with the next step.
 
 
     ```shell
-    export KFAPP=kfapp
-    export REGION=us-west-2
-    export AWS_CLUSTER_NAME=kubeflow-aws
+    export KFAPP=kfaws
+
+    kfctl init ${KFAPP} --config=${CONFIG} -V
+    cd ${KFAPP}
 
-    ${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform aws \
-    --awsClusterName ${AWS_CLUSTER_NAME} \
-    --awsRegion ${REGION}
+    kfctl generate all -V
+    kfctl apply all -V
     ```
 
+    * KFAPP - Use a relative directory name here rather than absolute path, such as `kfapp`. It will be used as eks cluster name.
+    * CONFIG - Path to the configuration file
 
-    * AWS_CLUSTER_NAME - Specify a unique name for your Amazon EKS cluster.
-    * KFAPP - Use a relative directory name here rather than absolute path, such as `kfapp`.
-    * REGION - Use the AWS Region you want to create your cluster in.
 
-1. Generate and apply platform changes.
+    __*Important!!!*__ By default, these scripts create an AWS Application Load Balancer for Kubeflow that is open to public. This is good for development testing and for short term use, but we do not recommend that you use this configuration for production workloads.
 
-    You can customize your cluster configuration, control plane logging, and private cluster endpoint access before you `apply platform`, please see [Customizing Kubeflow on AWS](/docs/aws/customizing-aws) for more information.
+    To secure your installation, Follow the [instructions](/docs/aws/authentication) to add authentication.
 
-    ```shell
-    cd ${KFAPP}
-    ${KUBEFLOW_SRC}/scripts/kfctl.sh generate platform
-    # Customize your Amazon EKS cluster configuration before following the next step
-    ${KUBEFLOW_SRC}/scripts/kfctl.sh apply platform
-    ```
 
-    > Note: If you see `no kind "ClusterConfig" is registered for version "eksctl.io/v1alpha5"` error, it means your eksctl version is low and not compatible with latest version. Please upgrade it and have another try.
+1. Wait for all the resources to become ready in the `kubeflow` namespace.
+    ```
+    kubectl -n kubeflow get all
+    ```
 
-1. Generate and apply the Kubernetes changes.
+1. Open Kubeflow Dashboard
```

Comment:
Does the command below actually open the dashboard? The existing guide has more info here, and I think people will need more info to figure out how to make the Kubeflow UI appear.

Reply:
I change to get service endpoint and it make more sense now.